tests/test_elog.py

```diff
@@ -96,5 +113,5 @@ def test_elog_from_registry(patch_webservice):
     assert HutchELog.from_registry() is elog1
     elog3 = HutchELog('TST', primary=True)
     assert HutchELog.from_registry() is elog3
-    for elog in (elog0, elog1, elog2, elog3):
-        assert elog in registry
+    for el in (elog0, elog1, elog2, elog3):
+        assert el in registry
```

Comment:
Seems random but actually avoids having a variable named the same as the module :+1: